### ISSUE TITLE
Issue with folder and file paths that don't have glob patterns

### ISSUE THREAD

**Issue by alan-agius4** [mrmlnc/fast-glob#9]
I have some instances where the glob pattern is not a glob at all and is a path to a directory/file.

Example
```
const glob = require("fast-glob");
glob.sync("node_modules/zone.js");
```

**Error**
```
​​Maximum call stack size exceeded​​
```

I expect that this doesn't break and return the path.

**Comment by mrmlnc** [comment]
Please, write about your environment? macOS, Linux, Windows?

**Comment by alan-agius4** [comment]
Windows 10

**Comment by mrmlnc** [comment]
Fixed in `1.0.1`.
